my-code/src/components/Thumbnail/styled.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import styled from "styled-components"
+
+type ThemeType = {
+  theme: {
+    color: {
+      grey: {
+        grey: string
+      }
+    }
+  }
+}
+
+const SmallEmptyThumbnail = styled.div`
```

Comment:
You can export directly the const here, instead of at the end of the file:

`export const SmallEmptyThumbnail = styled.div` `